common/guild_base.h

```diff
@@ -105,31 +153,36 @@ class BaseGuildManager
 		virtual void SendRankUpdate(uint32 CharID) = 0;
 		virtual void SendGuildDelete(uint32 guild_id) = 0;
 
-		uint32	DBCreateGuild(const char* name, uint32 leader_char_id);
```

Comment:
I know you're following existing patterns here but these don't need `DB` as a prefix, just describe what they do and then maybe we call a repository method underneath

Reply:
Updated them all and propagated throughout.